Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstoryStreamingAPI/GoogleChronicleBackstoryStreamingAPI.py

```diff
@@ -113,6 +131,14 @@ def build_http_client(self):
 ''' HELPER FUNCTIONS '''
 
 
+def remove_space_from_args(args):
+    """Remove space from args."""
+    for key in args.keys():
+        if isinstance(args[key], str):
+            args[key] = args[key].strip()
+    return args
```

Comment:
```suggestion
def remove_space_from_args(args):
    """
    Return a new dictionary with leading and trailing whitespace removed from all string values.

    :param args: Dictionary of arguments.
    :return: New dictionary with whitespace-stripped string values.
    """
    return {key: value.strip() if isinstance(value, str) else value for key, value in args.items()}
```

Both ways will work, however the original one will mutate the existing data where as this will return a new dictionary. Entirely up to you :)

Reply:
Hi @amshamah419, Due to this change, 4.1.0 release is failing for test connection and streaming fetch functionality.
In your suggested code, the dictionary is getting constructed from scratch. The original one has the [SmartGetDict](https://xsoar.pan.dev/docs/reference/api/common-server-python#smartgetdict) type, a dict that when called with get(key, default) will return the default passed value, even if there is a value of "None" in the place of the key. Example with built-in dict.

We have raised the new PR to handle this scenario. Please check it out. https://github.com/demisto/content/pull/37594

![image](https://github.com/user-attachments/assets/0c44c414-6ac7-42cc-b140-4797030a8348)